src/Repository/RepositoryStack.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Repository;
+
+/**
+ * @author Ener-Getick <egetick@gmail.com>
+ */
+class RepositoryStack
```

Comment:
Yes I saw that comment, but I don't understand
>interpedent repositories (for example, carsonbot could move the pull requests in the main repository). A stack would allow to fetch data from the sub repo

What do you mean by that?

Because it seems to me that this repo is an api for post request and each of them is bound to one repo.

If you're thinking of read-only sub like https://github.com/silexphp/Silex-Api, I guess only current users can tell if it might be a use case for it in the future.

In any case this could be done easily after #31 as well.

From this PR how would you handle the labels "hasPR" and "in progress" used for the docs and not in the core for example, I think it requires repo specific actions, right?